doc/formats.md

```
@@ -270,14 +325,15 @@ When auto-detecting format, the plugin uses this priority:
 1. **File extension** (`.lcov`, `.json`, `.xml`, `.gcda`, `.profdata`)
 2. **Content inspection** (for extensionless files)
    - Lines starting with `TN:`, `FN:`, `DA:` → LCOV
-  - Header `mode:` + profile ranges (`file.go:start,end ...`) → Go Coverprofile
-   - Lines starting with `{` → LLVM JSON
+    - Header `mode:` + profile ranges (`file.go:start,end ...`) → Go Coverprofile
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 3; Actual: 4

(MD005, list-indent)